src/components/ListItem/ListItem.vue

```diff
@@ -170,11 +170,15 @@
 	</listItem>
 	<listItem
 		:title="'Title of the element'"
+		:compact="true"
 		:counter-number=4
```

Comment:
Still invalid without quotes

Reply:
Actually, all the `counter-number` on this docs if possible :see_no_evil:

Reply:
Ha, hadn't noticed that. Done

Reply:
Sorry, I should have been more clear ahah, thanks!! :hugs: